internal/variables/variablesmap.gen.go

```diff
@@ -212,6 +212,207 @@ func (v RuleVariable) Name() string {
 	}
 }
 
+// CanBeSelected returns true if the variable supports selection (ie, `:foobar`)
+func (v RuleVariable) CanBeSelected() bool {
+	switch v {
```

Comment:
I don't think adding everything here makes sense. Just return true on those who can, and use the default otherwise.

Reply:
This is auto generated so I would not be concern about readability. @blotus could you do a quick benchmark on this matter i.e. adding everything or just true and all the rest on a default?

Reply:
I agree that readability does not really matter here.
I've updated the code generation to only generate the cases where true is returned.